lib/rubocop/cop/style/redundant_line_continuation.rb

```diff
@@ -180,7 +184,7 @@ def argument_newline?(node)
 
         def find_node_for_line(last_line)
           processed_source.ast.each_node do |node|
-            return node if node.respond_to?(:expression) && node.expression&.last_line == last_line
+            return node if same_line?(node, last_line)
```

Comment:
The previous version of this caused `find_node_for_line` to return the root node unless there was a node that responds to `expression`, which is only assignment nodes. This resulted in false negatives for cases where multiple continuations are found in the same `begin` node.